transpire/helpers/helm.py

```diff
@@ -95,6 +95,35 @@ def build_chart(
     if shutil.which("helm") is None:
         raise RuntimeError("You must install Helm to use this script.")
 
+    # make sure the current user has a place for us to cache stuff
+    home_cache = os.environ.get('XDG_CACHE_HOME', None)
+    if not home_cache:
+        raise EnvironmentError("Could not find an XDG_CACHE_HOME environment variable")
+    cache_dir = os.path.join(home_cache, 'ocf_helm')
```

Comment:
We should rename this to `transpire` instead of `ocf_helm`.

Reply:
done!